iotdb/src/main/java/org/apache/iotdb/db/query/executor/EngineExecutorWithoutTimeGenerator.java

```diff
@@ -44,7 +37,7 @@
 /**
  * IoTDB query executor with global time filter.
  */
-public class EngineExecutorWithoutTimeGenerator {
+public class EngineExecutorWithoutTimeGenerator extends ExecutorWithoutTimeGenerator {
```

Comment:
The class name is not clear.

Reply:
Change `ExecutorWithoutTimeGenerator` to `AbstractExecutorWithoutTimeGenerator `